src/server/dune

```diff
@@ -1,4 +1,4 @@
 (library
   (name httpkit_server)
   (public_name httpkit.server)
-  (libraries fmt logs httpaf uri))
+  (libraries fmt logs httpaf uri lwt))
```

Comment:
Could we remove this? We shouldn't depend on `lwt` for the core library 🤔 Or did you need to add this at any point?

Reply:
I think the compiler complained at some point about an "unbound module". I checked again and it's compiling without this: https://github.com/ostera/httpkit/pull/13/commits/c28d4db6ffae0d1b1e058927b538f5680ff9ad1f